packages/models/src/tenant/tenant.ts

```diff
@@ -68,6 +81,7 @@ export const TenantRevoker = z.object({
   expirationDate: z.coerce.date().optional(),
   extensionDate: z.coerce.date().optional(),
   id: TenantId,
+  delegationId: DelegationId.optional(),
```

Comment:
See the other comment about `rejectionReasons` in the descriptor - the same holds for these two `delegationId` new optional fields. Searching for places where the `verificationDate:` is set, I found a few examples where this could be missing (there are probably more):
- BFF [tenantApiConverter](https://vscode.dev/github/pagopa/interop-be-monorepo/blob/feature/pin-5126-capofila/packages/backend-for-frontend/src/api/tenantApiConverter.ts#L35-L36)
- tenant process [apiConverter](https://vscode.dev/github/pagopa/interop-be-monorepo/blob/feature/pin-5126-capofila/packages/tenant-process/src/model/domain/apiConverter.ts#L53)

Reply:
@Carminepo2 I see that verifiers/revokers are also used in the Agreement service/API. Could you double-check if it's correct that the delegationId is not present there or if it's missing?
https://vscode.dev/github/pagopa/interop-be-monorepo/blob/feature/pin-5126-capofila/packages/agreement-process/src/model/domain/apiConverter.ts#L104

Reply:
Good catch, added them [here](https://github.com/pagopa/interop-be-monorepo/pull/1175/commits/df94ef639f3ce0747e2c6d365c7bb0e3eef24330)